src/macros.jl

```diff
@@ -1222,11 +1278,11 @@ end
 
 Replaces `_MA.Zero` with a floating point `0.0`.
 """
-_replace_zero(::_MA.Zero) = 0.0
+_replace_zero(::_MA.Zero) = false
```

Comment:
I think we can fix this part to support returning the right `zero(value_type(model))`.

Reply:
See `_functionize` comment.